src/Common/src/Interop/Unix/System.Private.CoreLib.Native/Interop.MemReAlloc.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {
+        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_MemReAlloc")]
+        internal static extern IntPtr MemReAlloc(IntPtr ptr, UIntPtr newSize);
+        
+        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_MemGetAllocatedSize")]
+        internal static extern IntPtr MemGetAllocatedSize(IntPtr ptr);
```

Comment:
I can lookup the details in the source control history when I am in office next week. Either way, since it is not needed for coreclr, it should not be needed for corert either.